.github/workflows/versionci.yml

```
@@ -0,0 +1,20 @@
+name: Version CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.2.0", "3.2.7", "3.3.0", "3.3.7"]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
```

Should `bundler-cache` be `false` since we're testing across a few different Ruby versions?

```suggestion
          bundler-cache: false
```

---

@nicoledow We want to keep `bundler-cache: true`. Bundler automatically creates a separate cache for each Ruby version and gemfile.lock combination, so it won’t mix or conflict across different Ruby versions.

Without the cache, all gems would be re-installed on every run, which slows down the action significantly.